include/dali/core/tensor_view.h

```diff
@@ -650,6 +666,27 @@ TensorListView<StorageBackend, T, ndim> make_tensor_list(T *const *data,
   return { data, std::move(shape) };
 }
 
+template <typename StorageBackend, int ndim, typename T>
+TensorListView<StorageBackend, T, ndim> make_tensor_list(
+    const TensorView<StorageBackend, T, ndim> &tv) {
+  return {tv.data, {tv.shape}};
+}
+
+template <typename StorageBackend, int ndim, typename T>
+TensorListView<StorageBackend, T, ndim> make_tensor_list(
+    const std::vector<TensorView<StorageBackend, T, ndim>> &tvs) {
+  std::vector<TensorShape<ndim>> shapes(tvs.size());
+  for (size_t i = 0; i < tvs.size(); i++) {
+    shapes[i] = tvs[i].shape;
+  }
+  TensorListView<StorageBackend, T, ndim> tlv{nullptr, shapes};
+  tlv.data.resize(tvs.size());
+  for (size_t i = 0; i < tvs.size(); i++) {
+    tlv.data[i] = tvs[i].data;
+  }
```

Comment:
You don't need this intermediate vector:
```suggestion
  if (tvs.empty())
    return {};
  TensorListView<StorageBackend, T, ndim> tlv;
  tlv.resize(tvs.size(), tvs[0].dim());
  for (size_t i = 0; i < tvs.size(); i++) {
    tlv.shape.set_tensor_shape(i, tvs[i].shape);
    tlv.data[i] = tvs[i].data;
  }
```